README.md

```diff
@@ -21,8 +21,24 @@ export default function Page() {
 
 ## Setup
 
+### Install the package
+
 ```bash
 cd your-next-app
+
+# If your project is using NPM (the default for Next.js)
+npm install --save-dev @mux/next-video
+
+# If your project is using Yarn
+yarn add --dev @mux/next-video
+
+# If your project is using pnpm
+pnpm add --D @mux/next-video
```

Comment:
Is it really necessary to spell all these out, or can we just say something like "or your package manager of choice"? I feel like if you use yarn you know how to translate an npm install to it.

Reply:
it's pretty common for folks to call it out, especially if the expectation is to install it in `--dev` or something, which...now I'm wondering if that even really works for us since the component needs to exist in the build 😭